### DISCUSSION TITLE
여러 개발 문서 관리 위치 통일

### DISCUSSION THREAD

**Discussion by robinjoon** [woowacourse-teams/2024-devel-up#542] [백로그 논의]
#### 논의가 필요한 내용

현재 우리가 만드는 개발 문서가 깃허브 위키, 깃허브 이슈, 노션, 블로그, 구글 시트 등에 산재되어 있어요. 시간이 갈수록 어떤 문서가 어디에 있는지 찾아내기가 어려워지는 것 같아요. 용도에 맞게 문서를 정리할 필요가 있을 것 같아요.

블로그 역시 `~다`, `~해요` 등 문체를 통일해서 업로드하면 좋을 것 같아요.

#### 기타

우선순위는 높지만 급하지는 않은 것 같아요.

**Comment by Minjoo522** [comment]
> 현재 우리가 만드는 개발 문서가 깃허브 위키, 깃허브 이슈, 노션, 블로그, 구글 시트 등에 산재되어 있어요. 시간이 갈수록 어떤 문서가 어디에 있는지 찾아내기가 어려워지는 것 같아요. 용도에 맞게 문서를 정리할 필요가 있을 것 같아요.

제가 생각하기엔 우리 팀원들이 역시 지금도 용도에 맞게 잘 사용해주고 있는 것 같은데 어떨 때 어떻게 사용되고 있는지 한 곳에 정리해서 볼 수 있다면 편할것 같긴 합니다!

> 블로그 역시 ~다, ~해요 등 문체를 통일해서 업로드하면 좋을 것 같아요.

어차피 개인이 작성하는 문서이기 때문에 각자 자신의 개성에 맞게 작성해도 될 것 같습니다!
